engine/src/main/resources/assets/prefabs/player/player.prefab

```diff
@@ -74,11 +67,5 @@
         "translate": [-0.4, 0.45, 0.3],
         "scale": 0.35
   },
-  "GazeMountPoint": {
```

Comment:
does the mount point update before the game starts?

Reply:
Currently yes, it is updated upon player spawn with the value from the player settings. I want to touch the player settings and see what we can get rid of. 
I'd likely remove these settings, but keep the scale event upon player spawn to make sure the gaze mount point is correctly computed. I'd like to avoid managing default values in several places, thus, I'd like to get it out of the prefab...